scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 # shellcheck disable=SC1090,SC2086,SC2154,SC2034
 # ,SC2034,SC2143,SC2046,
-# Creators: gufmar, Scitz0
+# Creators: gufmar, Scitz0, Papacarp
 # 2020-05-19 cntools initial release (concept)
 # 2020-05-24 helper functions moved cnlibrary & configuration to env file
 
 ########## Global tasks ###########################################
 
 # get common env variables
-. "$(dirname $0)"/env
+. "$(dirname $0)"/env_guild
```

Comment:
Wouldnt this cause confusion between users? Not everyone using these script uses guild repo, and we'd hopefully rely on CNODE_HOME to distinguish between different networks

Reply:
I also think we should keep existing env name, can be used with other networks as well, not exclusive for guild network in my book.

Reply:
yep, that is an artifact of my testing.

Reply:
would it make sense to have one env file with [common] and [testnet: pHTN ... ] sections?

There could be one initial variable ACTIVE_TESTNET="pHTN" and then each testnet in a case section.
Then the prereqs curl command is always the same, and switching is quick and easy